index.bs

```diff
@@ -250,7 +250,7 @@ Report image Element Timing {#sec-report-image-element}
 <div algorithm="report image element timing">
     When asked to <dfn>report image element timing</dfn> given a triple (|element|, |imageRequest|, |loadTime|), a DOMHighResTimestamp |renderTime| and a {{Document}} |document|, perform the following steps:
 
-    1. Let |intersectionRect| be the value returned by the <a>intersection rect algorithm</a> using |element| as the target and viewport as the root.
+    1. Let |intersectionRect| be the value returned by the <a>intersection rect algorithm</a> using |element| as the target and |document|'s [=initial containing block=] as the root.
```

Comment:
Not sure if these are the same. If this element was on an iframe, was viewport the entire page viewport or the iframe's viewport?

Reply:
For an element in an iframe, `viewport` should be the iframe's bounds, and the element should be clipped to that for LCP calculations.

Reply:
Ok, makes sense. I guess you need a similar change in the text one?